models/Basketball_arena.java

```diff
@@ -0,0 +1,30 @@
+package models;
```

Comment:
Please move the models package to ua.lviv.iot, the same folder where your Main.java is.